#!/bin/sh /usr/share/dpatch/dpatch-run
## 07-init-fail.dpatch by Michael Meskes <meskes@debian.org>
##
## DP: Fix some failures in init script.

@DPATCH@

diff -Naurp virtualbox-ose-1.6.0-dfsg.orig/src/VBox/Installer/linux/vboxnet.sh.in virtualbox-ose-1.6.0-dfsg/src/VBox/Installer/linux/vboxnet.sh.in
--- virtualbox-ose-1.6.0-dfsg.orig/src/VBox/Installer/linux/vboxnet.sh.in	2008-04-30 14:05:52.000000000 +0000
+++ virtualbox-ose-1.6.0-dfsg/src/VBox/Installer/linux/vboxnet.sh.in	2008-05-09 09:28:11.000000000 +0000
@@ -49,6 +49,7 @@ fi
 if [ -z "$NOLSB" ]; then
     . /lib/lsb/init-functions
     fail_msg() {
+        [ -f "$VARFILE" ] && rm $VARFILE
         echo ""
         log_failure_msg "$1"
     }
